Makefile

```
@@ -117,7 +120,7 @@ deploy: manifests
 
 load-image: ## Load the operator image to a kind cluster
 load-image: container
-	./hack/load-image.sh $(IMAGE) $(VERSION)
+	./hack/load-image.sh "$(IMAGE) $(VERSION)"
```

Looks like image and version need to be separate parameters:
```suggestion
	./hack/load-image.sh "$(IMAGE)" "$(VERSION)"
```
https://github.com/openshift/contour-operator/blob/909d8a266eecb26d8cb06bddda6aefe20681212c/hack/load-image.sh#L8-L9